test/library/encryption/kms/k8s-mock-plugin/wrapper/main.go

```
@@ -0,0 +1,100 @@
+	raw := kmstesting.ReadAsset("k8s_mock_kms_plugin_configmap.yaml")
+	data := resourceread.ReadConfigMapV1OrDie(raw).Data
+
+	if err := os.WriteFile(defaultConfigPath, []byte(data["softhsm-config.json"]), 0644); err != nil {
+		return err
+	}
+	if err := os.MkdirAll(defaultTokensDir, 0755); err != nil {
+		return err
+	}
+
+	tokensB64 := data["softhsm-tokens.tar.gz.b64"]
+	cmd := exec.Command("sh", "-c", "base64 -d | tar xzf -")
```

golangci-lint (2.11.4)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)